driver/src/orderbook/streamed/state.rs

```diff
@@ -652,4 +691,93 @@ mod tests {
             2
         );
     }
+
+    #[test]
+    fn orderbook_batch_id() {
+        let mut state = state_with_fee();
```

Comment:
I was not aware of `state_with_fee` is this introduced here? I thought that constructors like this always took the form `State::new_with_fee`

Reply:
It is just a function that was already there from a previous test. If you open the whole file you can see it.